runtime/browser/ui/gtk2_ui.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef XWALK_RUNTIME_BROWSER_UI_GTK2_UI_H_
+#define XWALK_RUNTIME_BROWSER_UI_GTK2_UI_H_
+
+#include "ui/views/linux_ui/linux_ui.h"
+
+// Return the unique instance of class 'Gtk2UI'.
+extern views::LinuxUI* BuildGtk2UI();
+
+#endif  // XWALK_RUNTIME_BROWSER_UI_GTK2_UI_H_
```

Comment:
Ah, now I understood, you're keeping same file structure as `src/chrome/browser/ui/libgtk2ui/gtk2_ui.h`, then I'm okay as current.